velox/functions/prestosql/aggregates/tests/ArbitraryTest.cpp

```diff
@@ -196,5 +196,124 @@ TEST_F(ArbitraryTest, numericConstAndNulls) {
       "SELECT c0, first(c1), first(c2) FROM tmp group by c0");
 }
 
+TEST_F(ArbitraryTest, boolean) {
+  auto data = makeRowVector({
+      // Grouping key.
+      makeFlatVector<int64_t>({1, 1, 2, 2, 3, 3, 4, 4}),
+      // Input values: constant within groups.
+      makeFlatVector<bool>(
+          {true, true, false, false, true, true, false, false}),
+  });
+
+  auto expectedResult = makeRowVector({
+      makeFlatVector<int64_t>({1, 2, 3, 4}),
+      makeFlatVector<bool>({true, false, true, false}),
+  });
+
+  testAggregations({data}, {"c0"}, {"arbitrary(c1)"}, {expectedResult});
+
+  // Global aggregation.
+  auto plan = PlanBuilder()
+                  .values({data})
+                  .singleAggregation({}, {"arbitrary(c1)"})
+                  .planNode();
+
+  assertQuery(plan, "SELECT true");
```

Comment:
Shall we test the null case for boolean type?

Reply:
Sure. Added.